src/components/NewWorkspaceModal.js

```diff
@@ -161,18 +159,18 @@ const NewWorkspaceModal = Utils.withDisplayName('NewWorkspaceModal', ({
           value: namespace,
           onChange: ({ value }) => setNamespace(value),
           styles: { option: provided => ({ ...provided, padding: 0 }) },
-          options: _.uniq(_.map(({ projectName, invalidBillingAccount }) => ({
-            label: h(Fragment, [
-              invalidBillingAccount ?
+          options: _.map(({ projectName, invalidBillingAccount }) => ({
+            label: ((label = div({ style: { padding: 10 } }, [projectName])) => {
+              return invalidBillingAccount ?
                 h(TooltipTrigger, {
                   content: 'Workspaces may only be created in billing projects that have a Google billing account accessible in Terra',
```

Comment:
You can just make content `invalidBillingAccount && 'Workspaces...'` here, and remove the ternary altogether; a tooltip won't be rendered if content is falsy. Why did you make label an argument to this function if it's never called with arguments? I continue to not like the self-invoking function pattern.